plugins/maciVoting/hooks/useGetPollData.tsx

```diff
@@ -56,7 +56,7 @@ export const useGetPollData = (pollId?: string | bigint) => {
         now,
         voteEnded,
         disabled,
-        tallied: true,
+        tallied,
```

Comment:
This was set to always return true (I think it is because of debugging purposes). I made it return the real value